Spots-Mobile/src/components/Login.js

```diff
@@ -0,0 +1,109 @@
+// Login Screen JS code
+
+import React, { Component } from 'react';
+import { AppRegistry, StyleSheet, View, Image, Text, TextInput, TouchableOpacity, TouchableWithoutFeedback, Keyboard, KeyboardAvoidingView } from 'react-native';
```

Comment:
lets try to make this easier to read since this line got so long: 
```javascript
import {
  AppRegistry,
  Image,
  Keyboard,
  KeyboardAvoidingView,
  StyleSheet,
  Text,
  TextInput,
  TouchableOpacity,
  TouchableWithoutFeedback,
  View } from 'react-native';
```